xml/ha_migration.xml

```diff
@@ -169,19 +169,21 @@
    <itemizedlist>
     <listitem>
      <para>
-      Mixed clusters running on &productname; 11/&productname; 12 are
+      Mixed clusters running on &productname; 12/&productname; 15 are
```

Comment:
Just an idea: would it make sense to define some entities for the release numbers? I would have something like this in my mind:

```xml
<!ENTITY lastmajorver "12">
<!ENTITY majorver "15">
```

That could simplify that part and we don't have to deal with correct version numbers anymore. :wink:

Reply:
Thanks, but for the upgrade chapter, I prefer having the version numbers directly in the text. There are so many different ones to mention and if replacing them all by entities I fear we would sooner or later forget to update them accordingly.